src/services/targetApi.js

```diff
@@ -23,20 +35,27 @@ export const targetApi = createApi({
       query: (body) => {
         return { url: "/users/sign_in", method: "POST", body };
       },
-      transformResponse: (_, meta) => {
+      transformResponse: (res, meta) => {
         const { headers } = meta.response;
         return {
           access: headers.get("Access-Token"),
           client: headers.get("Client"),
           uid: headers.get("Uid"),
+          name: res.user.first_name,
         };
       },
     }),
+    deleteLogOut: builder.mutation({
+      query: (body) => {
+        return { url: "/users/sign_out", method: "DELETE", body };
```

Comment:
You can remove the `return` statement and make a one-liner:

```js
query: body => ({ url: "/users/sign_out", method: "DELETE", body })
```

And same with the other queries.